CHARTER.md

```diff
@@ -0,0 +1,257 @@
+# JSON Schema Org Charter
+<!-- This document is managed in the json-schema-org/.github GitHub repository. Please do NOT modify this file in another repository as changes may be overridden. -->
+
+<!-- While this document is being written, you can find the discussion to help determine what should be found here at https://github.com/json-schema-org/community/discussions/286 - This comment should be removed before merging -->
+
+## Section 0: Guiding Principles
+<!-- https://github.com/json-schema-org/community/discussions/286#discussioncomment-4391241 -->
+
+The JSON Schema project is part of the OpenJS Foundation which operates transparently, openly, collaboratively, and ethically. We strive to be open and transparent as much as is possible, and wish to enable anyone to interact and engage with any area of our work.
+
+Having no structure in place usually leads to one that is informal and undocumented, making it difficult to meet our own expectations of how we wish to operate. As such, we define the following charter which includes aspects of the governance model to which we subscribe and by which we operate.
+
+## Section 1: Scope
+<!-- https://github.com/json-schema-org/community/discussions/286#discussioncomment-4391250 -->
+
+JSON Schema aims to enable the confident and reliable use of the JSON data format. It does this primarily by providing specification documents which define a declarative language that allows annotation and validation of JSON documents.
+While JSON Schema's primary target is constraint-based data validation, it continues to be used across the whole stack, in different stages of a given project, and for purposes beyond its original design. We aim to enable these additional and emergent use cases.
+
+### 1.1: In-scope (optional)
```

Comment:
I assume removing these (optional) suffixes is part of the "removal of comments" checkbox right?

Reply:
Added this explicitly. Thanks for checking.